From 70f37ede687bfd3cfdecf0a83afff87733e98f98 Mon Sep 17 00:00:00 2001
From: Julien Olivain <julien.olivain@nxp.com>
Date: Tue, 15 Nov 2016 17:35:13 +0100
Subject: [PATCH 2218/5242] MLK-13471: fxls8471: add a symbol export to fix
 module build

commit  1f9ee266963130c8ee50f9f9382ffd33cc85460d from
https://source.codeaurora.org/external/imx/linux-imx.git

When CONFIG_SENSOR_FXLS8471=m build was failing due to missing
exported symbol. This patch export the missing symbol.

Signed-off-by: Julien Olivain <julien.olivain@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/input/misc/fxls8471.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/input/misc/fxls8471.c b/drivers/input/misc/fxls8471.c
index e77ef1b..ef7368b 100644
--- a/drivers/input/misc/fxls8471.c
+++ b/drivers/input/misc/fxls8471.c
@@ -49,6 +49,7 @@
 #define FXLS8471_BUF_SIZE		6
 
 struct fxls8471_data fxls8471_dev;
+EXPORT_SYMBOL(fxls8471_dev);
 
 static int fxls8471_position_setting[8][3][3] = {
 	{{0, -1, 0}, {1, 0, 0}, {0, 0, 1} },
-- 
1.7.9.5

